StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1130UseLambdaSyntax.cs

```diff
@@ -96,6 +96,28 @@ internal static ParameterListSyntax GetDelegateParameterList(ISymbol symbol, int
             return SyntaxFactory.ParameterList(SyntaxFactory.SeparatedList(syntaxParameters));
         }
 
+        internal static int FindArgumentIndex(SymbolInfo originalSymbolInfo, ArgumentSyntax argumentSyntax, BaseArgumentListSyntax argumentListSyntax)
+        {
+            // if delegate is passed as named argument of method try to find its position by argument name
+            if (originalSymbolInfo.Symbol.Kind == SymbolKind.Method
+                && originalSymbolInfo.Symbol is IMethodSymbol methodSymbol
+                && argumentSyntax.NameColon != null)
```

Comment:
💡 This (`NameColon`) check is the fast and common case; consider moving it to first in this condition